### DISCUSSION TITLE
What are these things that git added to the files I pushed?

### DISCUSSION THREAD

**Discussion by synthels** [community/community#23298] [New to GitHub]
<pre><code>package gr.logos.types;

&lt;&lt;&lt;&lt;&lt;&lt;&lt; HEAD // &lt;-------------- this
/**
 * @author Synthels
 */
=======
import gr.logos.tokens.Tokens;

/**
* @author Synthels
*/
&gt;&gt;&gt;&gt;&gt;&gt;&gt; 081b7cd... added some parsing // &lt;-------------- this
public class Types {
	public final static String INTEGER = "ΑΚΕΡΑΙΟΣ";
	public final static String UNSIG_INT = "ΘΕΤΙΚΟΣ_ΑΚΕΡΑΙΟΣ";
	public final static String STRING = "string";
	public final static String BOOLEAN = "bool";
	public final static String LIST = "ΛΙΣΤΑ";

	static String[] types = {"ΑΚΕΡΑΙΟΣ", "ΘΕΤΙΚΟΣ_ΑΚΕΡΑΙΟΣ", "string", "bool", "ΛΙΣΤΑ"};

	public static boolean isValidType(String token) {
		for(String s : types) {
			if(token.equals(s))
				return true;
			}
				return false;

		}
&lt;&lt;&lt;&lt;&lt;&lt;&lt; HEAD // &lt;-------------- this

	}
======= // &lt;-------------- this

	public static boolean isType(String str) {
		return (str.charAt(0) == '"' &amp;&amp; str.charAt(str.length() - 1) == '"') || Tokens.isNumber(str) || str.equals("true") || str.contentEquals("false");
	}

}
&gt;&gt;&gt;&gt;&gt;&gt;&gt; 081b7cd... added some parsing // &lt;-------------- this
</code></pre>

**Comment by lesleyanneb** [comment, marked as answer]
<p>Hi there,</p>
<p>Thank you for contributing to the GitHub community forum!</p>
<p>Those lines you’re seeing are conflict markers when you solve merge conflicts on the command line. Usually one would decide if you want to keep only your changes, keep only the other branch’s changes, or make a brand new change and then delete the conflict markers. The changes are separated by =======.</p>
<pre><code>&lt;&lt;&lt;&lt;&lt;&lt;&lt;, =======, &gt;&gt;&gt;&gt;&gt;&gt;&gt;
</code></pre>
<p>We have a guide that talks about this in more detail:</p>
<p><a href="https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line" rel="nofollow noopener">https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line</a></p>
<p>I hope that helps to explain. Please let us know if you need anything else!</p>
